controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -658,3 +683,26 @@ func (r *StorageConsumerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&rookCephv1.CephClient{}).
 		Complete(r)
 }
+
+func (c *cephClientFormat) generateHashForCephClient(storageConsumerName, claimID, cephUserType string) string {
+	c.StorageConsumerName = storageConsumerName
+	c.ClaimID = claimID
+	c.CephUserType = cephUserType
```

Comment:
To complete the prev comment:

```suggestion
	var c struct {
 		StorageConsumerName string
 		ClaimID string
 		CephUserType string
	}

	c.StorageConsumerName = storageConsumerName
	c.ClaimID = claimID
	c.CephUserType = cephUserType
```

Reply:
updated accordingly